src/app/view/components/_common/pattern-library/pattern-library.component.html

```diff
@@ -15,6 +15,29 @@ <h1>Hello World</h1>
     <h1>Hello World</h1>
 </dot-dialog>
 
+<h3 class="first">Single Row Selection</h3>
+<p-table [columns]="cols2" [value]="cars" selectionMode="single" [(selection)]="selectedCar1" dataKey="vin">
```

Comment:
Shouldn't we add here OUR listing table component?

Reply:
I preferred to add the component alone, so anyone can use it without needing the `ListingDataTableComponent`